src/unix/utime.c

```diff
@@ -70,7 +77,7 @@ double _al_unix_get_time(void)
    struct timespec now;
    double time;
 
-   _al_clock_gettime(CLOCK_REALTIME, &now);
```

Comment:
This would change the behavior of al_get_time. The documentation says: "Return the number of seconds since the Allegro library was initialised."

There is no room for interpreting that as skipping hibernation etc. Not saying anything about the other changes in this PR, but al_get_time is real time. It would potentially break a lot of code if that is changed.

Reply:
True that sounds like the correct one to me.